### ISSUE TITLE
Keeps uploading pictures making duplicate files {1)(2)(3)... 

### ISSUE THREAD

**Issue by izznogooood** [nextcloud/android#1670]
### Actual behaviour
Nextcloud uploads the same picture with the same file name (all of them) ... And keeps doing it. I had to Uninstall the app. I only noticed when my storage increased by 10GB over night. 

### Expected behaviour
Not that. 
 
### Steps to reproduce
1. Install app
2. Connect 
3. Picture1 picture1(1) picture(2)... You get the point. 

### Environment data
Android version:
7.0
Device model: 
Huawei P9

Stock or customized system:
Stock

Nextcloud app version:
Current

Nextcloud server version:
Current 

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by mario** [comment]
Please tell us what Nc Android version do you have installed.

**Comment by izznogooood** [comment]
Was 1.4.3

**Comment by mario** [comment]
Please try 2.0.0 RC9.

**Comment by izznogooood** [comment]
How? Where?

I only installed from the Play Store

**Comment by mario** [comment]
https://play.google.com/apps/testing/com.nextcloud.client

**Comment by mario** [comment]
This will soon be released as stable 2.0.0 :)
